spec/spec_helper.rb

```diff
@@ -1,2 +1,5 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'trix'
+require 'action_controller'
```

Comment:
Finally, this was the simplest way to get it all working, otherwise we'd need to check the rails version and require different files for Rails 4 and 5.